e2e/nx-misc/src/watch.test.ts

```diff
@@ -29,9 +34,19 @@ describe('Nx Commands', () => {
     runCLI(`generate @nx/js:lib ${proj1}`);
     runCLI(`generate @nx/js:lib ${proj2}`);
     runCLI(`generate @nx/js:lib ${proj3}`);
+    runCLI('daemon --start', {
+      env: {
+        NX_DAEMON: 'true',
+        NX_NATIVE_LOGGING: 'nx',
+        NX_PROJECT_GRAPH_CACHE_DIRECTORY: cacheDirectory,
+      },
+    });
```

Comment:
Um... daemon is already true I'd hope right?

Do we need the `NX_PROJECT_GRAPH_CACHE_DIRECTORY`?

And for the `NATIVE_LOGGING`, either flag it under `NX_VERBOSE_E2E` or remove it.